docker/Dockerfile

```diff
@@ -47,26 +47,32 @@ RUN rm -rf /opt/meshcentral/meshcentral/docker /opt/meshcentral/meshcentral/node
 
 ### STAGE 2 PRECOMPILE DEPS MODULE
 
-FROM alpine:3.22 AS dep-compiler
+### STAGE 2 PRECOMPILE SYSLOG MODULE
 
-RUN apk update && \
-    echo -e "----------\nINSTALLING ALPINE PACKAGES...\n----------"; \
-    apk add --no-cache --update \
-        bash gcc g++ jq make nodejs npm python3 tzdata
+FROM node:lts-trixie-slim AS syslog-compiler
 
-COPY --from=builder /opt/meshcentral/meshcentral /opt/meshcentral/meshcentral
+#RUN apk update && \
+RUN apt-get update && \
+    echo -e "----------\nINSTALLING DEBIAN PACKAGES...\n----------"; \
+    apt-get install -y \
+        bash gcc g++ jq make python3 tzdata
+#    apk add --no-cache --update \
+#        bash gcc g++ jq make python3 tzdata
+
+RUN mkdir -p /opt/meshcentral/meshcentral
 WORKDIR /opt/meshcentral/meshcentral
+RUN npm install modern-syslog
```

Comment:
you forgot telegram (check my line below)
and plz can you pin versions so things dont break in future :)